components/ComparisonTable.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <main class="storage-section pt-6 mt-6">
-    <div class="comparison-container sticky has-background-white">
+    <div class="comparison-container sticky has-background-white has-text-dark">
```

Comment:
I think it would be nice if the header had a dark background instead of light, as it would more clearly outline where the table starts. So I would either change the order of gray/white alternation, or even better have the header be a darker gray

Reply:
Maybe use `has-background-dark has-text-white`?

Reply:
Done!! Thank you, @mirestrepo!!